third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -207,6 +212,23 @@ The `google_sheets_options` block supports:
     that BigQuery will skip when reading the data. At least one of `range` or
     `skip_leading_rows` must be set.
 
+The `hive_partitioning_options` block supports:
+
+* `mode` (Optional) - When set, what mode of hive partitioning to use when
+    reading data. Two modes are supported. * AUTO: automatically infer
+    partition key name(s) and type(s). * STRINGS: automatically infer
+    partition key name(s). All types are Not all storage formats support hive
+    partitioning. Requesting hive partitioning on an unsupported format will
+    lead to an error. Currently supported formats are: JSON, CSV, ORC, Avro and Parquet.
+
+* `source_uri_prefix` (Optional) - When hive partition detection is requested,
```

Comment:
Lets quote the `gs://...` uris here, as it should make them easier to read. Or possibly link to the guide on source_uri_prefix? This seems like a pretty complicated concept that may be better explained by the docs than a small snippet in the description

Reply:
On second reading of this I'm not sure it makes sense. I see that the API docs have the same text, but I would prefer if we could clean it up.

Maybe something like this guide shows: https://cloud.google.com/bigquery/docs/hive-partitioned-queries-gcs#setting_hivepartitioningoptions_using_the_rest_api

"
Note that when hive_partitioning_options.mode is set to `CUSTOM`, you must encode the partition key schema within the `source_uri_prefix` by setting hive_partitioning_options.source_uri_prefix to `gs://bucket/path_to_table/{key1:TYPE1}/{key2:TYPE2}/{key3:TYPE3}`.

When hive partitioning is requested with either `AUTO` or `STRINGS` detection, the common prefix can be either of `gs://bucket/path_to_table` or `gs://bucket/path_to_table/`.
"